util/make_new_dif.py

```diff
@@ -178,7 +178,7 @@ def main():
         # Format autogenerated file with clang-format.
         try:
             subprocess.check_call(
-                ["./ci/bazelisk.sh", "run", "//quality:clang_format_fix", "--"] + out_files
+                ["./bazelisk.sh", "run", "//quality:clang_format_fix", "--"] + out_files
```

Comment:
@drewmacrae I changed this to call bazelisk directly to resolve the airgapped env. issue. Will this cause troubles somewhere else?

Reply:
LGTM, thanks